docs/04-documentation.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update documentation directly on GitHub.
+
+## Prerequisite
+
+If you don't already have a GitHub account, make sure to create one and log in with it.  
+You also have to be part of team [ECL doc editors](https://github.com/orgs/ec-europa/teams/ecl-doc-editors/).
+
+## Step 1: Open documentation file
+
+- Go to [ECL repository](https://github.com/ec-europa/europa-component-library)
+- Locate the component you wish to edit. Components are located in `framework/components`
+- Open the README.md file in that component (example: `framework/components/ecl-footers/README.md`)
+
+<img src="/assets/screenshots/git_edit.png" alt="GitHub edit" width="800" />
```

Comment:
Any reason why `<img>` is used instead of [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images)?

Reply:
Yes, to set the width

Reply:
Honestly, I feel it is not necessary to put a width on it. What's the added value?

Before I forget, please use Fractal's `path` helper: `{{ path '/assets/screenshots/git_edit.png' }}`

Reply:
Well I found them too big (longer scroll to get to next section). That's the main justification :)

Reply:
Then why not resizing the file itself? 🤔

Reply:
I may...